src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -217,20 +216,24 @@ public static void setJDBCLoggingDirectory() {
    *
    * @param config connector configuration
    */
-  static void validateProxySetting(Map<String, String> config) {
+  static Map<String, String> validateProxySetting(Map<String, String> config) {
```

Comment:
this method is used in SnowflakeSinkConnector (above file change) and in Utils. The error it throws is always caught, so I believe it is better to just return a list of the invalid params